### PR TITLE
fix(slider): avoid breaking thumb label on edges

### PR DESCRIPTION
**Pull request by jcfranco**
**Related Issue:** #9239

## Summary

This improves label handling for long thumb labels.

### CODE REVIEW

**Comment by jcfranco** [comment]
@josercarcamo FYI on some upcoming slider-related changes.

File: packages/calcite-components/src/components/slider/slider.tsx [hunk 1]
```diff
@@ -995,15 +995,14 @@ export class Slider
    * @private
    */
   private getHostOffset(leftBounds: number, rightBounds: number): number {
-    const hostBounds = this.el.getBoundingClientRect();
-    const buffer = 7;
```
Comment (eriklharper) [inline packages/calcite-components/src/components/slider/slider.tsx:999]: At first I wasn't sure about removing this buffer, but now that I look at the behavior, there's no more jumpiness in the label offset positioning any more.  I think the original intention to this buffer was to keep the label lined up exactly with the edge boundary of the track, but I like this better!  👍

File: packages/calcite-components/src/components/slider/slider.scss [hunk 2]
```diff
@@ -125,6 +125,8 @@
   transform: translate(var(--calcite-slider-thumb-x-offset), var(--calcite-slider-thumb-y-offset));
 
   .handle__label {
+    white-space: nowrap;
```
Comment (eriklharper) [inline packages/calcite-components/src/components/slider/slider.scss:128]: Gotta love 1-liners!

**Review by eriklharper — APPROVED** [review]
Should we add a screenshot test for this?

**Review by ashetland — APPROVED** [review]
🚀 🎉

**Pull request merged**